docs/build/workflow-snapshots.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Workflow Snapshots
+sidebar_label: Workflow Snapshots
+slug: /workflow-snapshots
+---
+
+Workflow snapshots capture and store the state of a workflow and job code at a specific point in time based on a save operation and/or run. Snapshots are helpful with debugging, auditing, and improving overall workflow performance.
+
+### When is a snapshot made?
+Snapshots are created in 2 ways:
+1. When a user saves changes to their workflow either through the canvas or the job editor
```

Comment:
What do you mean by "job editor"? Do you mean the `Inspector`? That's what we're calling it in [our current docs](https://docs.openfn.org/documentation/build/steps/step-editor) 
@ritazagoni